connectors/azure/README.md

```diff
@@ -19,3 +19,26 @@ docker run \
 ## Deploy with Helm-chart
 
 1. Set the required environment-variables.
+
+### Example deploy with Axorouter in cluster
+
+```bash
+make minikube-cluster
+make docker-build
+make minikube-load-image
+
+kubectl create namespace cloudconnectors
+kubectl create secret generic azure-event-hubs \
+  --from-literal=connection-string="<YOUR-AZURE-EVENT-HUBS-CONNECTION-STRING>" \
+  --namespace cloudconnectors \
+  --dry-run=client -o yaml | kubectl apply -f -
+
+helm upgrade --install --wait --namespace cloudconnectors cloudconnectors ./charts/cloudconnectors \
+  --set image.repository="axocloudconnectors" \
+  --set image.tag="dev" \
+  --set 'env[0].name=AXOROUTER_ENDPOINT' \
```

Comment:
FTR: it is possible to use `--set-json` in order to create the objects inline, not only field-by-field.